cmd/kubeadm/app/cmd/phases/reset/removeetcdmember.go

```diff
@@ -86,10 +96,17 @@ func getEtcdDataDir(manifestPath string, cfg *kubeadmapi.InitConfiguration) (str
 	klog.Warningln("[reset] No kubeadm config, using etcd pod spec to get data directory")
 
 	etcdPod, err := utilstaticpod.ReadStaticPodFromDisk(manifestPath)
-	if err != nil {
-		return "", err
+	if os.IsNotExist(err) {
+		// Fall back to use the default cluster config if etcd.yaml doesn't exist, this could happen that
+		// etcd.yaml is removed by the reset phase of cleanup-node.
+		cfg := &v1beta3.ClusterConfiguration{}
+		scheme.Scheme.Default(cfg)
+		return cfg.Etcd.Local.DataDir, nil
 	}
+	if err != nil {
+		klog.Warningf("[reset] Found error when loading etcd pod spec %v", err)
```

Comment:
`"[reset] Found error when loading etcd pod spec: %v"`
(added `:`)

github is not allowing me to use code "suggestions" for some reason